Sources/SecureXPC/Server/XPCServiceServer.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  XPCServiceServer.swift
+//  
+//
+//  Created by Alexander Momchilov on 2021-11-07.
+//
+
+import XPC
+
+internal class XPCServiceServer: XPCServer {
+	public static let service = XPCServiceServer()
```

Comment:
It doesn't make any difference since the class is `internal`, but make this `internal` as well for clarity?